api/database.js

```diff
@@ -35,6 +35,6 @@ if (!process.env.FIREBASE_CONFIG) {
             firebase
                 .database()
                 .ref(key)
-                .update(undefined),
+                .remove(),
```

Comment:
Although for now it might be enough – if it fixes the issue – maybe an `onComplete` function should be defined not only for [`remove`](https://firebase.google.com/docs/reference/js/firebase.database.OnDisconnect#remove) but also for [`set`](https://firebase.google.com/docs/reference/js/firebase.database.OnDisconnect#set). Let me know what you think.

Reply:
I believe you meant to link to this [remove](https://firebase.google.com/docs/reference/js/firebase.database.Reference#remove)

Reply:
Yes. Thanks for pointing that out, @peterhellberg.